setup.py

```diff
@@ -22,5 +22,6 @@
         "License :: OSI Approved :: GNU General Public License v3 (GPLv3)",
         "Operating System :: OS Independent",
     ],
-    install_requires=dependencies
+    install_requires=dependencies,
+    include_package_data=True,
```

Comment:
Is this to include the html, js, css?

Reply:
Yes, it includes everything in the `backends/statics` path. This is defined by the MANIFEST.in file.